go.mod

```diff
@@ -1,10 +1,11 @@
 module github.com/hyperledger/fabric-admin-sdk
 
-go 1.21
+go 1.23
```

Comment:
We should not move the minimum required Go version beyond 1.22 while 1.22 is still a supported Go version

Reply:
I suspect that having a 2-digit `go` directive and no `toolchain` directive will cause problems for consumers. I think we don't see that in our tests since the e2e tests are within the same module so do not exercise the API in the same way that an end user would.

Reply:
ok, I will update to go 1.22 later.

Reply:
The go.mod is only really supposed to be updated using the go CLI commands now. To update the Go version to 1.22 with no automatically added toolchain directive, you probably need `go get go@1.22.0 toolchain@none`. See the [Go Toolchains](https://go.dev/doc/toolchain) documentation; particularly the _Managing Go version module requirements with go get_ section.